pkg/manifests/manifests.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"io"
 
-	ingressv1alpha1 "github.com/openshift/cluster-ingress-operator/pkg/apis/ingress/v1alpha1"
+	operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
This can be grouped with `routev1` below, also from openshift/api.  Similar goes for many other files.  We can get this class of changes in a follow-up though.

Reply:
My intention was to avoid a bunch of reformatting diffs here — followup sounds good